tests/system/feature_store/test_feature_store.py

```diff
@@ -2454,6 +2454,33 @@ def test_deploy_ingestion_service_with_different_targets(
         for expected in expected_target_names:
             assert fset.get_target_path(expected) is not None
 
+    def test_feature_vector_with_all_features_and_label_feature(self):
+        feature_set = FeatureSet("fs-label", entities=[Entity("ticker")])
+        fs.ingest(feature_set, stocks)
+        expected = stocks.to_dict()
+        expected.pop("ticker")
+
+        fv = fs.FeatureVector("fv-label", ["fs-label.*"], "fs-label.name")
+        res = fs.get_offline_features(fv)
+
+        assert res is not None
+        assert res.to_dataframe().to_dict() == expected
+
+    def test_feature_vector_with_all_features_and_label_feature_two_feature_sets(self):
```

Comment:
```suggestion
    def test_feature_vector_with_all_features_and_label_feature_name_included_in_two_feature_sets(self):
```

A month from now, nobody will remember why this test is here or what's so special about it. So, I think the name needs to be more specific to the issue we're looking at, and please add a comment explaining what this tests.